sdk/alertsmanagement/Azure.ResourceManager.AlertsManagement/api/Azure.ResourceManager.AlertsManagement.netstandard2.0.cs

```diff
@@ -0,0 +1,644 @@
+        public static bool operator !=(Azure.ResourceManager.AlertsManagement.Models.MonitorService left, Azure.ResourceManager.AlertsManagement.Models.MonitorService right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class MonitorServiceDetails
+    {
+        internal MonitorServiceDetails() { }
+        public string DisplayName { get { throw null; } }
+        public string Name { get { throw null; } }
+    }
+    public partial class MonitorServiceList : Azure.ResourceManager.AlertsManagement.Models.ServiceAlertsMetaDataProperties
+    {
+        internal MonitorServiceList() { }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.AlertsManagement.Models.MonitorServiceDetails> Data { get { throw null; } }
+    }
+    public partial class MonthlyRecurrence : Azure.ResourceManager.AlertsManagement.Models.AlertProcessingRuleRecurrence
+    {
+        public MonthlyRecurrence(System.Collections.Generic.IEnumerable<int> daysOfMonth) { }
+        public System.Collections.Generic.IList<int> DaysOfMonth { get { throw null; } }
+    }
+    public partial class RemoveAllActionGroups : Azure.ResourceManager.AlertsManagement.Models.AlertProcessingRuleAction
+    {
+        public RemoveAllActionGroups() { }
+    }
+    public partial class ServiceAlertEssentials
+    {
+        public ServiceAlertEssentials() { }
+        public string AlertRule { get { throw null; } }
+        public Azure.ResourceManager.AlertsManagement.Models.AlertState? AlertState { get { throw null; } }
+        public string Description { get { throw null; } set { } }
+        public bool? IsSuppressed { get { throw null; } set { } }
+        public string LastModifiedBy { get { throw null; } }
+        public System.DateTimeOffset? LastModifiedOn { get { throw null; } }
+        public Azure.ResourceManager.AlertsManagement.Models.MonitorCondition? MonitorCondition { get { throw null; } }
+        public System.DateTimeOffset? MonitorConditionResolvedOn { get { throw null; } }
+        public Azure.ResourceManager.AlertsManagement.Models.MonitorService? MonitorService { get { throw null; } }
+        public Azure.ResourceManager.AlertsManagement.Models.ServiceAlertSeverity? Severity { get { throw null; } }
+        public Azure.ResourceManager.AlertsManagement.Models.ServiceAlertSignalType? SignalType { get { throw null; } }
+        public string SmartGroupId { get { throw null; } }
+        public string SmartGroupingReason { get { throw null; } }
+        public string SourceCreatedId { get { throw null; } }
+        public System.DateTimeOffset? StartedOn { get { throw null; } }
+        public string TargetResource { get { throw null; } set { } }
+        public string TargetResourceGroup { get { throw null; } set { } }
+        public string TargetResourceName { get { throw null; } set { } }
+        public string TargetResourceType { get { throw null; } set { } }
```

Comment:
Is this the ARM ResourceType we know? If not, shall we rename it to something like TargetResourceKind?

Reply:
this is a value of `enum`